lib/theme/index.js

```diff
@@ -80,6 +80,64 @@ export const generateTheme = ({
       },
     },
 
+    danger: {
+      backgroundColor: colors.red[600],
```

Comment:
In the [design system](https://www.figma.com/file/N4Xbl652BhzOutXmzhcrLGch/DS-06-%2F-Atoms?node-id=14%3A1) the following colors have been defined for the `danger` button:

```suggestion
      backgroundColor: colors.red[500],
```